lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -128,6 +131,75 @@ class ReportUIFeatures {
     dropdown.addEventListener('click', this.onExport);
   }
 
+  _setupThirdPartyFilter() {
+    const pageTLDPlusOne = new URL(this.json.finalUrl).origin.split('.').slice(-2).join('.');
+
+    // get all tables with a text url
+    /** @type {Array<HTMLTableElement>} */
+    const tables = Array.from(this._document.querySelectorAll('.lh-table'));
+    tables.filter(el => el.querySelector('td.lh-table-column--url'))
+      .filter(el => /** @type {Element} */ (el.closest('.lh-audit')).id !== 'uses-rel-preconnect')
+      .forEach((el, index) => {
```

Comment:
lets call this 'tableEl` since we're dealing with a few different elements in here.